lib/completionItem.ts

```diff
@@ -78,9 +75,9 @@ export function fromCompletionItem(
     result.filterText = completionItem.filterText
   }
 
-  if (completionItem.insertTextRules === CompletionItemInsertTextRule.InsertAsSnippet) {
+  if (completionItem.insertTextRules === 4 satisfies monaco.languages.CompletionItemInsertTextRule.InsertAsSnippet) {
     result.insertTextFormat = 2
```

Comment:
I feel like if we use `value satisfies monaco.*` syntax for all Monaco enums, the same pattern should be applied to all language server enums.